src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -119,6 +119,39 @@ def test_aks_create_and_update_with_managed_nat_gateway_outbound(self, resource_
                 'networkProfile.natGatewayProfile.managedOutboundIpProfile.count', 2),
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='eastus')
+    def test_aks_update_outbound_from_slb_to_natgateway(self, resource_group, resource_group_location):
```

Comment:
Queued [pipeline](https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=62809553&view=results) to generate recording.

Reply:
attached the recording files. Thank you !